code/modules/lighting/lighting_system.dm

```diff
@@ -131,8 +151,8 @@
 #else
 	distance = max(abs(x - L.__x), abs(y - L.__y))
 #endif
-	return ( L.cap ? L.cap : LIGHTING_CAP ) * (L.radius - distance) / L.radius
-//LIGHTING_CAP == strength for now
+
+	return Clamp(L.cap * (L.radius - distance) / L.radius, 0, LIGHTING_CAP)
```

Comment:
only one thing calls this.

the only reason it's not inlined, is so turfs can override how light is applied to them.

other wise i would have stuffed this in the add effect proc.